### ISSUE TITLE
No way to change avatar size

### ISSUE THREAD

**Issue by piotrkulpinski** [timber/timber#1730]
### Expected behavior
I would like to be able to change the size of the avatar pulled by `post.author.avatar`.

### Actual behavior
The avatar property always pulls the default gravatar size of 96x96.

### Steps to reproduce behavior
Use `post.author.avatar` anywhere in the single.twig

### What version of WordPress, PHP and Timber are you using?
WordPress 4.9.6, PHP 7.2.5, Timber 1.7.1

### How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)
WP.org

--- 
Looks like the method is just pulling the deafult avatar without allowing to set any arguments for `get_avatar_url` function:
https://github.com/timber/timber/blob/master/lib/User.php#L148

**Comment by piotrkulpinski** [comment]
Now, I know I can do `post.author.avatar.src | resize('128x128')` but it won't stretch the image if base size is 96px.

**Comment by palmiak** [comment]
Hi

True - that is a problem - I always add:
```php
function user_avatar( $size = '' ) {
        $args['size'] = $size;
        return get_avatar_url( $this->id, $args );
}
```

in Class that extends TimberPost so I can `post.user_avatar(500)`. It should be a easy thing to do - maybe try making a PR?

**Comment by jarednova** [comment]
@palmiak submitted a fix for this on #1928. Merged and will be released early next week!
